DESCRIPTION

```diff
@@ -28,9 +28,6 @@ Suggests:
     tibble,
     testthat (>= 3.0.0)
 Config/testthat/edition: 3
-LinkingTo: 
-    cpp11
-SystemRequirements: C++11
```

Comment:
Why this removal?

Reply:
OK, that makes sense. No need to move it, seems like more effort than it's worth!